Touch events

irenesmith opened the pull request:
A checklist to help your pull request get merged faster:
- Added BCD for Touch events to Document
- Added BCD for Touch events to Element

This is part refactoring touch events - see [ZenHub #950](https://app.zenhub.com/workspaces/mdn-sprint-board-5ab3d23fd0f4ea53b0022a61/issues/mdn/sprints/950)

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12”
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+      "touchcancel_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchcancel_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchcancel_event",
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+      "touchend_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchend_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchend_event",
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+      "touchmove_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchmove_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchmove_event",
```

vinyldarkscratch commented on api/Document.json:
```diff
@@ -7327,6 +7327,214 @@
+      "touchstart_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchstart_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchstart_event",
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+      "touchcancel_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchcancel_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchcancel_event",
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+      "touchend_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchend_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchend_event",
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+      "touchmove_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchmove_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchmove_event",
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+      "touchstart_event": {
+        "__compat": {
+          "description": "<code>webglcontextrestored</code> event",
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/Document/touchstart_event",
```
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/touchstart_event",
```

vinyldarkscratch commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "6"
```
```suggestion
              "version_added": "12"
```

jpmedley commented on api/Element.json:
```diff
@@ -5110,6 +5110,214 @@
           }
         }
       }
```
This line needs a comma.

irenesmith replied:
Added. Thanks!

irenesmith commented:
> 
> 
> Thanks for this, @irenesmith ! I fixed the merge conflicts and the lint errors.
> 
> The "description" is wrong for these events. Also, could you point to where the compat data itself comes from, and why you believe it's accurate (at least as accurate as it's reasonable to ascertain)?

I have updated the descriptions.

The BCD data came from the information that was on the event pages before I refactored them but I also confirmed the information with the specifications.
